packages/markdown-support/src/codeblock.ts

```diff
@@ -1,43 +1,44 @@
 import { visit } from 'unist-util-visit';
+import type { Node } from 'unist';
+import type { Element } from 'hast';
+import type { Code } from 'mdast';
 
 /**  */
 export function remarkCodeBlock() {
-  const visitor = (node: any) => {
-    const { data, meta } = node;
-    let lang = node.lang || 'html'; // default to html matches GFM behavior.
+  return function (tree: Node) {
```

Comment:
If you accept a `Root` (from `mdast`) here, then `visit` can already “magically” narrow down that `'code'` will result in only `Code` nodes!
`unist.Node` is pretty vague, where possible, I’d prefer using the more specific `mdast` implementations of that abstract interface!

Reply:
Amazing 🙌 Works like a charm, great job typing the unified ecosystem!

Reply:
Addressed in 3392ebb